cmd/operator-opamp-bridge/go.mod

```diff
@@ -6,89 +6,90 @@ require (
 	github.com/go-logr/logr v1.2.4
 	github.com/oklog/ulid/v2 v2.1.0
 	github.com/open-telemetry/opamp-go v0.7.0
-	github.com/open-telemetry/opentelemetry-operator v1.51.0
```

Comment:
We should retract this version, it's what we have as latest in go.pkg.dev right now.